storage/ndb/src/kernel/vm/SimulatedBlock.hpp

```diff
@@ -1804,13 +1802,13 @@ class alignas(NDB_CL) SimulatedBlock :
 #define NUM_LQHKEYREQ_COUNTS 4
 #define NUM_SCAN_FRAGREQ_COUNTS 1
 #define MAX_LDM_THREAD_GROUPS_PER_RR_GROUP 8
-#define MAX_RR_GROUPS ((MAX_NDBMT_QUERY_THREADS + \
+#define MAX_RR_GROUPS ((MAX_NDBMT_QUERY_WORKERS + \
                         (MIN_QUERY_INSTANCES_PER_RR_GROUP - 1)) /  \
                       MIN_QUERY_INSTANCES_PER_RR_GROUP)
 #define MAX_DISTRIBUTION_WEIGHT 16
 #define MAX_NUM_DISTR_SIGNAL \
           (MAX_DISTRIBUTION_WEIGHT * MAX_QUERY_INSTANCES_PER_RR_GROUP)
-#define MAX_DISTR_THREADS (MAX_NDBMT_LQH_THREADS)
+#define MAX_DISTR_THREADS (MAX_NDBMT_QUERY_WORKERS)
```

Comment:
The number of lqh workers isn't equal with query workers all the time when ldm is not configured as 0. Here should we use MAX_NDBMT_LQH_WORKERS instead to keep the compatibility.

Reply:
MAX_DISTR_THREADS is the number of threads that can assist in read queries (== Query workers)